app/src/features/surveys/view/survey-animals/SurveyAnimalsTable.tsx

```diff
@@ -72,16 +73,30 @@ export const SurveyAnimalsTable = ({
       )
     },
     {
-      field: 'deployments',
-      headerName: 'Device ID',
+      field: 'current_devices',
+      headerName: 'Current Devices',
       flex: 1,
-      renderCell: (params) => (
-        <Typography>
-          {params.value?.length
-            ? params.value?.map((device: IAnimalDeployment) => device.device_id).join(', ')
-            : 'No Device'}
-        </Typography>
-      )
+      valueGetter: (params) => {
+        const currentDeploys = params.row.deployments?.filter(
+          (device: IAnimalDeployment) => !device.attachment_end || moment(device.attachment_end).isAfter(moment())
+        );
+        return currentDeploys?.length
+          ? currentDeploys.map((device: IAnimalDeployment) => device.device_id).join(', ')
+          : 'No Devices';
+      }
+    },
+    {
+      field: 'previous_devices',
+      headerName: 'Previous Devices',
```

Comment:
Nit: Maybe previous devices could also be revised. Expired Devices or Inactive Devices?